### ISSUE TITLE
Events not working, And want to add hash in url only on tabs

### ISSUE THREAD

**Issue by erpankajsood** [flatlogic/bootstrap-tabcollapse#71]
As mentioned in the doc, Events not working?
-     show-tabs.bs.tabcollapse - triggered before tabs component is shown
-     shown-tabs.bs.tabcollapse - triggered after tabs component is shown
-     show-accordion.bs.tabcollapse - triggered before accordion component is shown
-     shown-accordion.bs.tabcollapse - triggered after accordion component is shown

`
$('#myTab').on('shown-accordion.bs.tabcollapse', function(){
    alert('accordion is shown now!');
});
`

Basically i need to add hash on the url in tabs. I am adding extra script for this but not working, its working on tabs but accordion creating issue on close.

I am trying also to remove **data-toggle="tab"** in the tabs and run by js. Then accordion is not working.

Suggest me how to fix this. Thanks :)
